test/workers/test-child-process.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+require('../common');
+
+require('./index.js').runTestsInsideWorker([
```

Comment:
Might be better, if possible, to grab all files from a readdir that match the `test-${subsystem}` prefix for each of these and explicitly blacklist specific files known to not work. That way new tests will run in worker tests too, by default.